radio/src/gui/common/stdlcd/radio_sdmanager.cpp

```diff
@@ -105,13 +105,14 @@ void onSdManagerMenu(const char * result)
     memcpy(reusableBuffer.sdmanager.originalName, line, sizeof(reusableBuffer.sdmanager.originalName));
     char * ext = getFileExtension(line, SD_SCREEN_FILE_LENGTH+1);
     if (ext) {
+      size_t len = strlen(ext);
```

Comment:
is it really needed to call `strlen` here? if not I will prefer the simple substraction (but with your code on 2 lines)